app/pods/publications/publication/publishpreview/controller.js

```diff
@@ -290,10 +298,20 @@ export default class PublicationPublishPreviewController extends Controller {
   @action
   async markPreviewActivityDone(previewActivity) {
     this.showLoader = true;
+    const openStatus = await this.store.findRecord('activity-status', CONFIG.ACTIVITY_STATUSSES.open.id);
     const closedStatus = await this.store.findRecord('activity-status', CONFIG.ACTIVITY_STATUSSES.closed.id);
-    previewActivity.status = closedStatus;
-    previewActivity.endDate = moment().utc();
-    await previewActivity.save();
+    const previewActivityActivityStatus = await previewActivity.get('status');
+
+    if (previewActivityActivityStatus.id === closedStatus.id) {
+      previewActivity.status = openStatus;
+      previewActivity.endDate = undefined;
```

Comment:
Ik zou hier ipv `undefined` , `null` gebruiken. Ik heb hier net even wat over opgezocht en kan me wel vinden in de denkwijze van de `most upvoted persoon` https://stackoverflow.com/questions/461966/why-is-there-a-null-value-in-javascript

`undefined` houden voor speciale gevallen en/of waardes die gedeclareerd zijn maar nog geen assignment gehad hebben. Als we die redenering doortrekken naar dit, dan zou hier `null` moeten staan want je had bv al een endDate kunnen hebben.